From 16dd4bb4b2d9f728850725bf05ba4e429483450a Mon Sep 17 00:00:00 2001
From: Watson Xu <xuhuashan@gmail.com>
Date: Fri, 3 Mar 2017 14:01:54 +0800
Subject: [PATCH 09/10] change configuration file path for yxgtech board

yxgtech boards needs the configuration file to be stored
at /etc/config directory.
---
 src/ipcam-runtime.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ipcam-runtime.cpp b/src/ipcam-runtime.cpp
index fc9ce67..6494eb0 100644
--- a/src/ipcam-runtime.cpp
+++ b/src/ipcam-runtime.cpp
@@ -51,7 +51,7 @@ IpcamRuntime::IpcamRuntime
 #endif
 {
 #ifdef HAVE_LIBCONFIG_SUPPORT
-	_config_name = SYSCONFDIR "/" PACKAGE_NAME ".conf";
+	_config_name = SYSCONFDIR "/config/" PACKAGE_NAME ".conf";
 	try {
 		_config.readFile(_config_name.c_str());
 	}
-- 
2.9.3

